cli/parser/options/flag_form/flag_form.go

```diff
@@ -0,0 +1,69 @@
+package flag_form
+
+type Form byte
+
+const (
+	// The standard form of a flag; for example: "--name"
+	Standard Form = iota
```

Comment:
Should this be `iota << 1` ? (I guess the code might be correct without this change, since the tests seem pretty exhaustive, but it seems like the intention might have been to define these as one-hot bit vectors, since you're doing some bitmasking below)